src/components/Codeblock.tsx

```diff
@@ -231,30 +231,11 @@ const codeTheme = {
   },
 }
 
-const getValidChildrenForCodeblock = (child) => {
```

Comment:
Not needed since in all of our use cases, we receive a `string` as the child. Forced this on the `IProps` definition.